src/software/backend/wifi_backend.cpp

```diff
@@ -15,13 +15,24 @@ WifiBackend::WifiBackend()
                     boost::bind(&WifiBackend::receiveWorld, this, _1),
                     DynamicParameters->getAIControlConfig()->getRefboxConfig(),
                     DynamicParameters->getCameraConfig()),
-      ssl_proto_client(boost::bind(&Backend::receiveSSLWrapperPacket, this, _1),
-                       boost::bind(&Backend::receiveSSLReferee, this, _1))
+      ssl_proto_client(
+          boost::bind(&Backend::receiveSSLWrapperPacket, this, _1),
+          boost::bind(&Backend::receiveSSLReferee, this, _1),
+          SSL_VISION_DEFAULT_MULTICAST_ADDRESS, SSL_VISION_MULTICAST_PORT + 1,
```

Comment:
I had to change the ports here because otherwise the `ssl_proto_client` and `network_client` conflict with one another because they both try bind to the network sockets. They are redundant so this currently doesn't affect functionality, and will be removed once GrSim is removed and SensorFusion is integrated right after.

Reply:
If they'll be removed once GrSim is removed and SensorFusion is integrated, it'd be best to leave a comment to this effect.

Reply:
left a comment and linked to an issue.

Reply:
actually no longer an issue since we don't need to bind to the `sender_endpoint`